examples/billing/get_billing_setup.py

```diff
@@ -37,47 +35,41 @@ def main(client, customer_id, page_size):
         'billing_setup.payments_account_info.secondary_payments_profile_id '
         'FROM billing_setup')
 
-    results = ga_service.search(customer_id, query=query, page_size=page_size)
+    response = ga_service.search_stream(customer_id, query=query)
 
     try:
         # Use the enum type to determine the enum name from the value.
-        billing_setup_status_enum = (
-            client.get_type('BillingSetupStatusEnum', version='v3')
-                .BillingSetupStatus)
+        billing_setup_status_enum = client.get_type(
+            'BillingSetupStatusEnum', version='v3').BillingSetupStatus
 
         print('Found the following billing setup results:')
-        for row in results:
-            billing_setup = row.billing_setup
-            payments_account_info = billing_setup.payments_account_info
-            print('Billing setup with ID "%s", status "%s", '
-                  'payments_account "%s", payments_account_id "%s", '
-                  'payments_account_name "%s", payments_profile_id "%s", '
-                  'payments_profile_name "%s", '
-                  'secondary_payments_profile_id "%s".'
-                  % (billing_setup.id.value,
-                     billing_setup_status_enum.Name(billing_setup.status),
-                     billing_setup.payments_account.value,
-                     payments_account_info.payments_account_id.value,
-                     payments_account_info.payments_account_name.value,
-                     payments_account_info.payments_profile_id.value,
-                     payments_account_info.payments_profile_name.value,
-                     payments_account_info.secondary_payments_profile_id.value))
-    except google.ads.google_ads.errors.GoogleAdsException as ex:
-        print('Request with ID "%s" failed with status "%s" and includes the '
-              'following errors:' % (ex.request_id, ex.error.code().name))
+        for batch in response:
+            for row in batch.results:
+                billing_setup = row.billing_setup
+                payments_account_info = billing_setup.payments_account_info
+                print(f'Billing setup with ID {billing_setup.id.value}, '
+                      f'status "{billing_setup_status_enum.Name(billing_setup.status)}", '
+                      f'payments_account "{billing_setup.payments_account.value}" '
+                      f'payments_account_id "{payments_account_info.payments_account_id.value}", '
+                      f'payments_account_name "{payments_account_info.payments_account_name.value}", '
+                      f'payments_profile_id {payments_account_info.payments_profile_id.value}, '
+                      f'payments_profile_name "{payments_account_info.payments_profile_name.value}", '
+                      f'secondary_payments_profile_id "{payments_account_info.secondary_payments_profile_id.value}".')
```

Comment:
Nit: this is a _little_ tough to read, would it be worth setting some of these values to variables above the print statement so that the printed string is a easier to parse visually?

Reply:
Used a shorter prefix variable name. Looking forward to getting rid of `.value`